css/index.css

```diff
@@ -1,3 +1,158 @@
 /* Example of @import use
 This imports the `reset.css` file */
 @import './reset.css';
+
+* {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+  list-style: none;
+  text-decoration: none;
+}
+
+body {
+  font-family: "Poppins", sans-serif;
+  background-color: #e5e5e5;
+}
+
+:root {
+  --primaryColor: #6070ff;
+}
+
+.firstsection {
+  background-color: #fff;
+  height: 100vh;
+  border-radius: 0 0 0 150px;
+  padding: 35px 24px;
+}
+
+/* NAVIGATION BAR */
+nav {
+  display: flex;
+  justify-content: space-between;
+  padding: 0 24px;
+}
+
+.nav-checkbox {
+  display: none;
+}
+
+.logo-container {
+  color: #6070ff;
+  font-weight: 700;
+  font-size: 18px;
+  line-height: 20px;
+  cursor: pointer;
+}
+
+.hamburger-btn {
+  color: #67798e;
+  font-size: 1.5rem;
+  cursor: pointer;
+}
+
+/* END OF NAVIGATION BAR */
+
+/* INTRODUCTION */
+.introduction {
+  display: flex;
+  height: 85vh;
+  justify-content: center;
+  align-items: center;
+}
+
+.ellipse11 {
+  position: fixed;
+  top: 128.64px;
+  left: 0;
+}
+
```

Comment:
Please let us try to make our code [dry](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself) as much as possible. We have so many properties like the `position: fixed` and more repeated across various class names, we can do the following instead:

- Either create a generic class name like `.fixed { position: fixed }` below and pass it in our HTML class attribute or
- combine the classes that have that properties like below
```css
.ellipse11,
.ellipse5,
.ellipse3,
.ellipse8 {
  position: fixed;
}
```


Please let's correct this and improve on this in subsequent milestone and projects